### DISCUSSION TITLE
starstruck

### DISCUSSION THREAD

**Discussion by ghost** [Schweinepriester/github-profile-achievements#417] [Q&A] [answered]
Leave links to your repositories that you need to star 🌟

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by kshitij-halankar** [reply]
@number0x0001 done https://github.com/kshitij-halankar/TravelR

**Reply by aarongeo1** [reply]
@number0x0001 @kshitij-halankar done https://github.com/aarongeo1/NUM-CLASSIFY

**Comment by HHNebula** [comment]
done https://github.com/HHNebula/HHNebula

**Comment by 41pha1** [comment]
Done
https://github.com/41pha1/MIDI-Converter

**Comment by aarongeo1** [comment]
done https://github.com/aarongeo1/NUM-CLASSIFY

**Comment by drk1rd** [comment]
Done!
https://github.com/drk1rd/drk1rd

**Comment by callqh** [comment, marked as answer]
Done, bro. Do it for me, too :)
https://github.com/liuqh0609/note

**Reply by ArielMAJ** [reply]
@callqh I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by bm4578** [reply]
> 完成了，兄弟。也给我做:) https://github.com/liuqh0609/note

Have finished  https://github.com/bm4578/docker.git

**Comment by bm4578** [comment]
Have finished ,Can you help me? https://github.com/bm4578/docker.git
